application_sdk/constants.py

```diff
@@ -30,6 +30,8 @@
 # Application Constants
 #: Name of the application, used for identification
 APPLICATION_NAME = os.getenv("ATLAN_APPLICATION_NAME", "default")
+#: Name of the deployment, used to distinguish between different deployments of the same application
+DEPLOYMENT_NAME = os.getenv("ATLAN_DEPLOYMENT_NAME", None)
```

Comment:
Can this be part of the same deployment secret store

Reply:
default value should be `local`. When deployed on atlan it will be `atlan` and a custom value when deployed on customer's end